src/components/TrackTile/TrackTile.tsx

```diff
@@ -73,14 +74,16 @@ export function TrackTile({
 }
 
 const defaultStyles = createStyles('TrackTile', (theme) => ({
-  titleText: {
-    margin: theme.spacing.medium,
+  title: {
+    marginVertical: -8,
   },
+  titleText: {},
```

Comment:
I feel like I have seen this in a couple of places. Not in this PR but I feel like it would be good to add `title` to our list of `TextStyle` type prefixes and then switch any `title` styles to be `titleContainer` or something like that. `titleText` just reads kind of clunky to me.

Reply:
That is a good thought. I wonder how intuitive `title` would be that it automatically becomes a `TextStyle` type though.  I like that we have sufixes where `somethingText` becomes a `TextStyle` type and `somethingView` becomes a `ViewStyle` type.  Same with `somethingImage` and `ImageStyle`.  These map directly.  The default being that if it doesn't end with any of those three it is a `ViewStyle`.  Adding more rules than that for common use cases like `title` might get confusing, what about `header` or `name` for instance.  Could we accomplish the same thing by getting more specific in our naming when there is ambiguity?  I could just change the above to be `titleView` and `titleText` for example.

Reply:
Good points, let's just stick with it for now then 👍